lib/base-compiler.ts

```diff
@@ -83,8 +84,8 @@ export class BaseCompiler {
     public lang: Language;
     protected compileFilename: string;
     protected env: any;
-    // Note that this can also return undefined and boolean | number, but we still need properties.js to ts
-    protected compilerProps: (key: string, defaultValue?: string) => string;
+    // Note that this can also return undefined and boolean | number, but we still need properties.ts to ts
```

Comment:
nitpick (probably from my OG change...so I blame me) -> this comment makes no sense any more. Maybe just remove it?

Reply:
Good catch, I'll go ahead and remove the comment